contrib/opentimelineio_contrib/adapters/tests/test_ale_adapter.py

```diff
@@ -98,6 +99,37 @@ def test_ale_read2(self):
             ]
         )
 
+    def test_ale_read3(self):
```

Comment:
Would you mind adding a check in the tests to make sure the CDL data comes through correctly?

Reply:
Thanks God you actually asked me to do that! :D
I found that I had swapped the offset and power values.
I added the cdl test. Let me know.